src/name_core.jl

```diff
@@ -162,26 +162,36 @@ This is the type of name combination used for binary array operations.
 Where the dimensions of both arrays must be the same.
 """
 function unify_names(names_a, names_b)
-    # 0-Allocations if inputs are the same
-    # 0-Allocation, if has a `:_` see  `@btime (()->unify_names((:a, :b), (:a, :_)))()`
-
+    # @btime (()->unify_names((:a, :b), (:a, :_)))()
     names_a === names_b && return names_a
```

Comment:
can this line be moved into `try_unify_names` ?
It should constant fold alway, right?